scripts/api.py

```diff
@@ -37,9 +38,25 @@ async def model_list():
 
     @app.get("/controlnet/module_list")
     async def module_list(alias_names: bool = False):
-        _module_list = external_code.get_modules(alias_names)
-        print(_module_list)
-        return {"module_list": _module_list}
+        _module_list = external_code.get_modules(False)
+        _module_list_alias = external_code.get_modules(True)
+        _module_detail = {}
+        
+        _output_list = _module_list if not alias_names else _module_list_alias
+        for index, module in enumerate(_output_list):
+            if _module_list[index] in preprocessor_sliders_config:
+                _module_detail[module] = {
+                    "sliders": preprocessor_sliders_config[_module_list[index]]
+                }
+            else:
+                _module_detail[module] = {
+                    "sliders": []
+                }
+        
+        return {
+            "module_list": _output_list,
+            "module_detail": _module_detail
+        }
```

Comment:
this looks like a good contribution. you think it would make sense to expose the module_detail values through the external_code api as well?

Reply:
I had never used the external_code api before...Let me have a look.